pgbouncer/README.md

```diff
@@ -11,43 +11,64 @@ The PgBouncer check is included in the [Datadog Agent][1] package, so you don't
 
 ### Configuration
 
-Edit the `pgbouncer.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][7]. See the [sample pgbouncer.d/conf.yaml][2] for all available configuration options:
+To capture PGBouncer metrics you need to install the Datadog Agent on your PGBouncer server.
 
-```
-init_config:
+1. Create a read-only datadog user with proper access to your PostgreSQL Server. Start psql on your PostgreSQL database and run:
 
-instances:
-  - host: localhost
-    port: 15433
-    username: <YOUR_USERNAME>
-    password: <YOUR_PASSWORD>
-#   tags:
-#     - env:prod
-  - database_url: postgresql://<DB_USER>:<DB_PASS>@<DB_HOST>:<DB_PORT>/dbname?sslmode=require
-#   tags:
-#     - role:main
-```
+    ```
+    CREATE USER datadog WITH PASSWORD '<YOUR_PASSWORD>';
+    GRANT SELECT ON pg_stat_database TO datadog;
 
```

Comment:
This block should be separated in two. The last two lines are verification commands that the user as correctly added.